android/src/main/java/com/oney/gcm/GcmModule.java

```diff
@@ -106,6 +118,7 @@ public void onReceive(Context context, Intent intent) {
                 params.putString("dataJSON", bundleString);
 
                 sendEvent("remoteNotificationReceived", params);
+                abortBroadcast();
```

Comment:
is it this line that prevents the GcmBroadcastReceiver to also receive the intent (and start again the react app in background) ?
A BroadcastReceiver defined in AndroidManifest always come after the one programatically registered ?